### DISCUSSION TITLE
Host Streamlit App's on Deta

### DISCUSSION THREAD

**Discussion by Sven-Bo** [deta/cloud-docs#423] [General] [answered]
Is it possible to host & run [Streamlit](https://streamlit.io/) app's on deta?
I assume it is not possible because I am getting the error `Error: failed to update dependencies: Request entity too large` when trying to deploy the app (& install the dependency `streamlit`) on deta.

**Comment by AyushSehrawat** [comment, marked as answer]
Probably the dependencies crossing the allowed size. 

```
The total upload size of your source code and assets is limited to 250 MB.
Dependencies (pip, npm, etc) also can't exceed a combined size of 250mb.
```

**Reply by Sven-Bo** [reply]
Thanks for the prompt reply. Yes, I thought so too

**Comment by abdelhai** [comment]
hi @Sven-Bo, the issue is what Ayush mentioned. that said, we're working on removing this limitation in the coming months

**Reply by Sven-Bo** [reply]
Hi Mustafa, thanks for your reply and the info! 👍

**Reply by AyushSehrawat** [reply]
@abdelhai will there be changes to payload limit too?

**Reply by abdelhai** [reply]
this is planned but might only be available in the first half of next year
